### ISSUE TITLE
Gazebo installation

### ISSUE THREAD

**Issue by SebastianBitsch** [adeeb10abbas/ros2-docker-dev#5]
Thanks for this nice resource! 

**TLDR; what should I include in the docker file to get Gazebo running with ROS2?**

I am having trouble making it work with Gazebo.
If I try to include an installation of Gazebo in the Dockerfile 
```
RUN apt-get install -y ros-humble-ros-gz && \
    rm -rf /var/lib/apt/lists/*
```
I get the error 
```
 => ERROR [ros_dev_env 6/7] RUN apt-get install -y ros-humble-ros-gz &&     rm -rf /v  0.2s
------                                                                                               
 > [ros_dev_env 6/7] RUN apt-get install -y ros-humble-ros-gz &&     rm -rf /var/lib/apt/lists/*:
0.172 Reading package lists...
0.197 Building dependency tree...
0.199 Reading state information...
0.201 E: Unable to locate package ros-humble-ros-gz
------
```
I follow the steps you outline in the short video, and build the container using:
```
ros_dev robotautonomy ~/pathto/ros2_ws/src
```

As far as I am aware ```ros-humble-ros-gz``` isn't available for arm, which seems to indicate that the container isn't building with the nonvc-1 image (see image below). I might however be misunderstanding something as I am no expert in docker. Any help on how you made it work would be greatly appreciated, and let me know if there is anything else you need from me, thanks.

<img width="857" alt="Skærmbillede 2024-02-20 kl  20 50 04" src="https://github.com/adeeb10abbas/ros2-docker-dev/assets/72623007/82e5b10d-e4b4-40e5-8394-6135216c77a7">

**Comment by adeeb10abbas** [comment]
Hi there! 

Thanks for your kind words. The reason it doesn't work is that, arm builds for gazebo aren't yet available for `apt-get` install. So, I believe you will have to clone the repo yourself and build it. 

Apart from that I found this comment (and also the parent thread which discusses this exact issue useful) https://github.com/gazebosim/gazebo-classic/issues/3236#issuecomment-1331053760

Since, I think this is beyond the scope of this project, I will close this issue. Although feel free to ask me any other question you may have. I haven't had time to work on it, so if you find ways to improve it I'd be more than happy to merge them in! Thanks!
